### PR TITLE
LPS-135713 Use predefined value when value is an empty string

### PR DESCRIPTION
**Pull request by alinedoleron**
Related Issue: https://issues.liferay.com/browse/LPS-135713

### CODE REVIEW

**Comment by liferay-forms** [comment]
ci:test:sf

**Comment by liferay-forms** [comment]
Hi, you don’t need to run the CI commands anymore, I’ll do it for you and let you know if any issue happens.

**Comment by liferay-forms** [comment]
ci:test:relevant

**Comment by alinedoleron** [comment]
Hi, @miltonmc. After investigating with @eli-rlima, we noticed that for some reason the richtext value on builder side comes as an empty string, so we replaced the `??` operator by `||`. It's really hard to debug because we need to type fast, and it's a intermittent bug.

**Comment by liferay-forms** [comment]
Hey @alinedoleron, can you please check if these failures are related to your changes? Thanks

**Comment by eli-rlima** [comment]
ci:reevaluate:1317401_253

**Comment by liferay-forms** [comment]
Hey @alinedoleron, can you please check if these failures are related to your changes? Thanks

File: modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-field-type/src/main/resources/META-INF/resources/RichText/RichText.es.js [hunk 1]
```diff
@@ -46,7 +46,7 @@ const RichText = ({
 		}
 	}, [editingLanguageId, editorRef]);
 
-	const currentValue = value ?? predefinedValue;
+	const currentValue = value || predefinedValue;
```
Comment (markocikos) [inline modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-field-type/src/main/resources/META-INF/resources/RichText/RichText.es.js:49]: Hey @alinedoleron 

This does not fix the issue. The root cause of the bug is that `value` becomes  the same as `predefinedValue`, after `change` event. Checking if it's `null` or falsy is not enough. After `value` changes to truthy, further changes in `predefinedValue` are ignored. 

Please see solution in https://github.com/liferay-forms/liferay-portal/pull/874

**Comment by eli-rlima** [comment]
ci:test:relevant

File: modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-field-type/src/main/resources/META-INF/resources/RichText/RichText.es.js (hunk 1 above)
Comment (eli-rlima) [reply modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-field-type/src/main/resources/META-INF/resources/RichText/RichText.es.js:49]: Hey @markocikos! 

@alinedoleron is in PTO until friday. I will see this now. Thanks for your review!

**Comment by eli-rlima** [comment]
Hey, @miltonmc!

This PR is blocked, I'm reviewing the solution.

**Comment by eli-rlima** [comment]
Resolved in #921.

**Comment by eli-rlima** [comment]
ci:close

**Pull request closed without merging**